### PR TITLE
docs: add "Tutorials" section with new Edge AI Anomaly Detection tutorial

### PR DESCRIPTION
**Pull request by mattdibi**
This PR adds a "Tutorial" section in the Kura documentation. In addition it adds the the Edge AI Anomaly Detection tutorial leveraging the Triton Inference Server integration.

### CODE REVIEW

**Comment by mattdibi** [comment]
@pierantoniomerlino @marcellorinaldo can one of you guys take a look at this? To @pierantoniomerlino could be interesting the Anomaly Detection tutorial part since we did the presentation together. To @marcellorinaldo could be relevant the documentation part: I added a plugin for rendering the Notebook inside MkDocs.

**Review by marcellorinaldo — COMMENTED** [review]
Screenshot 2022-11-08 at 14 36 07

There is some odd rendering.

File: docs/tutorials/AD-EdgeAI.ipynb [hunk 1]
```diff
@@ -0,0 +1,2379 @@
...
+  {
+   "cell_type": "markdown",
+   "id": "5f536ae1",
+   "metadata": {},
+   "source": [
+    "## Overview\n",
+    "\n",
+    "This document contains the code and the instructions for our EclipseCON 2022 Talk: \"_How to Train Your Dragon and Its Friends: AI on the Edge with Eclipse Kura&trade;_\"\n",
+    "\n",
+    "This notebook can also be viewed and ran on [Google Colab](https://colab.research.google.com/github/mattdibi/eclipsecon-edgeAI-talk/blob/master/notebook/AD-EdgeAI.ipynb).\n",
```
Comment (marcellorinaldo) [inline docs/tutorials/AD-EdgeAI.ipynb:20]: Do you plan to keep this link alive?

Comment (mattdibi) [reply docs/tutorials/AD-EdgeAI.ipynb:20]: Yup. It's just a wrapper around the link to my repository: as long as the repo is up the Colab link will work.

**Comment by mattdibi** [comment]
> Screenshot 2022-11-08 at 14 36 07
> 
> There is some odd rendering.

It works fine for me:

image

How did you build the sources?

**Comment by marcellorinaldo** [comment]
> > Screenshot 2022-11-08 at 14 36 07
> > There is some odd rendering.
> 
> It works fine for me:
> 
> image
> 
> How did you build the sources?

mkdocs serve

**Comment by marcellorinaldo** [comment]
> > Screenshot 2022-11-08 at 14 36 07
> > There is some odd rendering.
> 
> It works fine for me:
> 
> image
> 
> How did you build the sources?

It works just on Chrome

**Review by marcellorinaldo — APPROVED** [review]

**Comment by pierantoniomerlino** [comment]
@mattdibi Both in Firefox and Chrome the page is ok. Only the formulas are not correctly rendered.

**Comment by mattdibi** [comment]
> Only the formulas are not correctly rendered.

~I'll fix that with a dedicated PR~

**Edit**: @pierantoniomerlino On the online page the formulas are rendered correctly. 🥳

**Comment by marcellorinaldo** [comment]
@mattdibi We can backport this to dock-release-5.2 since it does not involve new functionalities

**Comment by mattdibi** [comment]
@marcellorinaldo That's a good idea. I can't add the label to trigger the backport job though since I - *sigh* - don't have the permissions to do so...

**Pull request merged**